packages/e2e-tests/specs/experimental/block-transforms.test.js

```diff
@@ -153,13 +153,8 @@ describe( 'Block transforms', () => {
 	} );
 
 	describe( 'correctly transform', () => {
-		beforeAll( async () => {
-			await createNewPost();
-		} );
-
 		beforeEach( async () => {
-			await setPostContent( '' );
-			await page.click( '.editor-post-title .editor-post-title__block' );
+			await createNewPost();
```

Comment:
It might slow down the execution of those tests. It needs to be confirmed whether the total execution time on Travis doesn't increase.

Reply:
It will be slower for sure. But slower is better than failing all the time?

Reply:
Most other tests start from a new post as well, that's nothing new.

Reply:
I agree, it may be slower than it was previously, but this is a pattern we adopt in many other tests.

I would actually very much like to see an improvement to the implementation of `createNewPost` where we try to artificially reset the state of the editor to an initial state, rather than perform a full reload. But that can be considered a separate task.

Reply:
This one is quite unique, this test suite alone runs for 5 minutes on Travis.

Reply:
I agree, that's something we should explore. Ideally it's almost identical to a page refresh where other browser state is reset, such as scroll position and I don't know what else. :)

Reply:
I thought there was an issue, but I couldn't find one, so I created it at #18668.

To @gziolo's last point, it would be a bit disappointing if this is run many times and is already a significant time-to-execute. Is the change to the other file sufficient for a fix?

Otherwise, I might still be okay with a longer runtime if it might at least serve as encouragement to finally implement #18668.

Reply:
No, the other change is not sufficient. :(

Reply:
In general, this particular test suite is semi-automated so it's execution time grows whenever a new block is added or new block transformation is introduced. The Group block has also a huge impact as it gets applied to every block. In addition, a test is created for nearly every added block variation to the list of test fixtures.

Reply:
@gziolo I'm aware of all this, but still, I think it's better to have slow passing tests than fast randomly failing tests. This is the best solution I can find for now. I'm not sure about others, but for me, these tests fail in almost every build and it's annoying to have to restart the failing jobs all the time, after which I also have to restart deploying the playground. It think time is generally also lost when reviewers (myself included) see a failing Tarvis build and might think that a PR is not ready yet.

Reply:
My point goes rather in the direction that the effort to maintain and execute those tests is significantly higher than having a few smoke tests confirming that this features works at the cost of not having all combinations covered.

Reply:
I think @jorgefilipecosta added them originally? I'm not sure if it's worth it or not.

Reply:
I think these tests, proved useful and caught some bugs with transforms in the past. But now transforms are mostly stable and rarely change and given the complications, they are giving us I guess we can remove them and then include some specific tests for some transforms.

Reply:
@ellatrix, would you prefer to update this PR to remove the tests instead, or I can submit a different PR that removes them?

Reply:
Ok, let's remove them then.

Reply:
@jorgefilipecosta I removed them in this PR.

Reply:
> But now transforms are mostly stable and rarely change and given the complications, they are giving us I guess we can remove them and then include some specific tests for some transforms.

Yes, we definitely want to keep them but for a limited set of blocks 👍 Let's make sure we add a test suite in a follow-up PR. We should have at least text blocks (List, Quote, Paragraph), media blocks (Image, Gallery, Cover) and the Group block covered.

@jorgefilipecosta thinking about it a bit more, would it be possible to recreate this test suite by creating 3 sections (text, media, group) and listing fixtures for the source of blocks and run checks on the expected snapshots?